sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -455,6 +456,22 @@ object DateTimeUtils {
     (MICROSECONDS.toSeconds(localTimestamp(microsec, timeZone)) % 60).toInt
   }
 
+  /**
+   * Returns seconds, including fractional parts, multiplied by 1 000. The timestamp
```

Comment:
`1 000` -> `1000`.

Reply:
Why is it better? I copy-pasted it from PostgreSQL docs.

Reply:
Let me ask you in this way, @MaxGekk .
1. If you think this is better, why not Java style `1_000_000` ? (AFAIK, this is supported since JDK7).
2. And, does Apache Spark have `1 000 000` or `1 000` in our code?
3. In this PR, I don't find any explanation why you introduce this new style in Apache Spark community in this PR.
4. Finally, this is Apache Spark doc, not `PostgreSQL` doc. A blind `copy&paste` is unable to justify that you thought this is better.